ci/pipelines/toolbox-deploy-ecr.yml

```diff
@@ -0,0 +1,102 @@
+resources:
+  - name: toolbox-deploy-ecr
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master 
+      paths:
+        - ci/pipelines/toolbox-deploy-ecr.yml
+  # Github Releases
+  - name: git-toolbox-staging-release
+    type: git
+    icon: github
+    source: &github-release-source
+      uri: https://github.com/alphagov/pay-toolbox
+      branch: master
+      tag_regex: "alpha_staging-2-(.*)"
+  # ECR registry resource
+  - name: ecr-registry-test
+    type: dev-registry-image
+    icon: docker
+    source:
+      repository: govukpay/toolbox
+      aws_access_key_id: ((readonly_access_key_id))
+      aws_secret_access_key: ((readonly_secret_access_key))
+      aws_session_token: ((readonly_session_token))
+      # Need to create the role for the concourse user to assume in the test account
+      aws_role_arn: arn:aws:iam::((pay_aws_staging_account_id)):role/concourse
+      # Hardcode the test account registry ID for now. Needs to be a string, not a number
+      aws_ecr_registry_id: "((pay_aws_staging_account_id))"
+      aws_region: eu-west-1
+
+resource_types:
+  # Custom resource type - this has been merged to the
+  # https://github.com/concourse/registry-image-resource repo,
+  # however is not yet in a stable release. Using this custom
+  # resource type allows us to support the ECR registry_id parameter
+  # This can be removed once registry_id is in stable release of 'registry-image'.
+  - name: dev-registry-image
+    type: registry-image
+    source:
+      repository: concourse/registry-image-resource
+      username: ((docker-username))
+      password: ((docker-password))
+      tag: dev
+
+jobs:
+  - name: update-toolbox-test-ecr-pipeline
+    plan:
+      - get: toolbox-deploy-ecr
+        trigger: true
+      - set_pipeline: toolbox-deploy-ecr
+        file: toolbox-test-ecr/ci/pipelines/toolbox-test-ecr.yml
+  - name: toolbox-staging-ecr
+    plan:
+      - get: git-toolbox-staging-release
+        trigger: true
+      # Temporarily fetch image from Dockerhub until Concourse can build its own
+      - task: pull-toolbox-from-dockerhub
```

Comment:
As discussed offline, we'll just fetch the image from the test ECR repo, rather than from DockerHub.